src/assets/scripts/client/InputController.js

```diff
@@ -332,104 +268,82 @@ export default class InputController {
         }
     }
 
+    // TODO: This does hardly anything of value. It should probably just be removed to simplify things.
     /**
      * @for InputController
      * @method onKeydownHandler
      * @param event {jQuery Event}
      * @private
      */
     onKeydownHandler(event) {
-        // For firefox see: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
-        const is_firefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
-
-        if (!GameController.game_paused()) {
-            this.$commandInput.focus();
-        }
-
-        if (event.which === KEY_CODES.ESCAPE) {
-            if (prop.tutorial.open) {
-                this._tutorialView.tutorial_close();
-            } else if ($(SELECTORS.DOM_SELECTORS.AIRPORT_SWITCH).hasClass(SELECTORS.CLASSNAMES.OPEN)) {
-                UiController.ui_airport_close();
-            }
-        }
-
-        if (!prop.tutorial.open) {
-            return;
-        }
-
-        if (event.which === KEY_CODES.PAGE_UP) {
-            this._tutorialView.tutorial_prev();
-            event.preventDefault();
-        } else if (event.which === KEY_CODES.PAGE_DOWN) {
-            this._tutorialView.tutorial_next();
-            event.preventDefault();
-        }
+        // // For firefox see: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
+        // const is_firefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
+        //
+        // if (!GameController.game_paused()) {
+        //     this.$commandInput.focus();
+        // }
+        //
+        // if (event.which === KEY_CODES.ESCAPE) {
+        //     if (prop.tutorial.open) {
+        //         this._tutorialView.tutorial_close();
+        //     } else if ($(SELECTORS.DOM_SELECTORS.AIRPORT_SWITCH).hasClass(SELECTORS.CLASSNAMES.OPEN)) {
+        //         UiController.ui_airport_close();
+        //     }
+        //
+        //     this.selectAircraft();
+        // }
     }
 
+    // TODO: Is this really needed??
```

Comment:
yes.

this is the method that gives us access to the _current_ value of the command input.  

hmm, I see what you're getting at though, changing how this works would definitely be a new issue.

Reply:
Yeah it's more of a question of rethinking how, or even if, we store the contents of the text field as a variable. Now I think it's in the UI controller and the prop, and both are used, with multiple places where it's being manually parsed to determine the selected aircraft etc. Would be a new issue.